topi/python/topi/x86/conv2d.py

```diff
@@ -702,33 +413,17 @@ def traverse(op):
                 data_pad = data
                 data = data_pad.op.input_tensors[0]
 
-            kh, kw = kernel_size if isinstance(kernel_size, (tuple, list)) else \
-                (kernel_size, kernel_size)
+            kh, kw = kernel_size if isinstance(kernel_size, (tuple, list)) \
+                else (kernel_size, kernel_size)
```

Comment:
We don't need kernel_size in the argument list. We can get it from `kernel` in L407